### PR TITLE
fix TS compatibility / add Glint support

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
## Changes

- Adds a `template-registry.d.ts` so that we can have ember-scoped-css work with Glint without custom definitions

## For review

You'll probably want to collapse the test directories first:

image

### CODE REVIEW

File: ember-scoped-css/template-registry.d.ts
Comment (NullVoxPopuli) [inline ember-scoped-css/template-registry.d.ts:1]: Here along with the package.json change is the actual fix

File: ember-scoped-css/package.json [hunk 1]
```diff
@@ -11,9 +11,14 @@
   "files": [
     "src",
     "dist",
-    "addon-main.cjs"
+    "addon-main.cjs",
+    "template-registry.d.ts"
   ],
   "exports": {
+    "./template-registry": {
+      "types": "./template-registry.d.ts",
```
Comment (NullVoxPopuli) [inline ember-scoped-css/package.json:19]: the actual fix

**Comment by NullVoxPopuli** [comment]
Seems maybe there is a real problem. Hmm

**Review by mattmcmanus — APPROVED** [review]

**Comment by NullVoxPopuli** [comment]
Looks like this accidentally got blocked by: https://github.com/soxhub/ember-scoped-css/issues/108

**Pull request merged**